streamer/src/test/utils/postgres/postgresQueries.js

```diff
@@ -0,0 +1,147 @@
+const {environment} = require('../../../environment')
+const DB_SCHEMA = environment.DB_SCHEMA
+const totalStakeThreshold = 1000000
+const totalRewardThreshold = 1000000
+const totalRewardPointsThreshold = 20
+
+
+async function getQuery(postgres, queryString) {
```

Comment:
This function seems to be totally unnecessary as it's just an additional level of abstraction that messes.

Reply:
IMO, that's reducing the number of lines for each query in functions

Reply:
First, decreasing or increasing number of strings in the code is not the purpose itself. 
Second, this function itself is just a proxy wrapper for the native .query method of pg. If you switch it inside the functions where you call it, nothing will change. But it's just an additional abstraction layer that decreases readability of the code bringing no value. The native pg.query does exactly the same with throwing it's own error that is more clear.